packages/kotti-ui/source/index.ts

```diff
@@ -120,76 +120,74 @@ import { KtTranslationContext } from './next/kotti-translation'
 export * from './next/kotti-translation'
 export * from './types'
 
-const components = {
-	KtAccordion,
-	KtActionbar,
-	KtAvatar,
-	KtAvatarGroup,
-	KtBanner,
-	KtBreadcrumb,
-	KtButton,
-	KtButtonGroup,
-	KtCard,
-	KtCheckbox,
-	KtCol,
-	KtComment,
-	KtCommentInput,
-	KtDateInput,
-	KtDatePicker,
-	KtDrawer,
-	KtDropdown,
-	KtDropdownButton,
-	KtDropdownMenu,
-	KtField,
-	KtFieldDate,
-	KtFieldDateRange,
-	KtFieldDateTime,
-	KtFieldDateTimeRange,
-	KtFieldMultiSelect,
-	KtFieldNumber,
-	KtFieldPassword,
-	KtFieldRadioGroup,
-	KtFieldSingleSelect,
-	KtFieldText,
-	KtFieldTextArea,
-	KtFieldToggle,
-	KtFieldToggleGroup,
-	KtForm,
-	KtFormControllerList,
-	KtFormControllerObject,
-	KtFormSubmit,
-	KtHeading,
-	KtInlineEdit,
-	KtInput,
-	KtInputNumber,
-	KtLine,
-	KtModal,
-	KtNavbar,
-	KtPagination,
-	KtPopover,
-	KtPopoverItem,
-	KtRadio,
-	KtRadioGroup,
-	KtRow,
-	KtSingleSelect,
-	KtStep,
-	KtSteps,
-	KtSwitch,
-	KtTable,
-	KtTableColumn,
-	KtTableProvider,
-	KtTableConsumer,
-	KtTableColumnsStateMixin,
-	KtTheme,
-	KtToaster,
-	KtTranslationContext,
-	KtUserMenu,
-}
-
 export default {
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any
 	install(Vue: any) {
-		for (const component of Object.values(components)) Vue.use(component)
+		for (const component of [
```

Comment:
noice

Reply:
Was worried that this would be considered a side-effect so I inlined it, just to be safe